pages/instructorFileTransfer/instructorFileTransfer.sql

```diff
@@ -14,7 +14,8 @@ FROM
   questions AS q
 WHERE
   q.uuid = $uuid
-  AND q.course_id = $course_id
+  AND q.course_id = $course_id -- leave this the same, right? In the long run, we want to allow people to make a copy
```

Comment:
Correct, this should stay the same for now.

Reply:
Should I leave a TODO comment here? It seems like a really long off feature at this point given everything else we are trying to do.

Reply:
I would lean toward not bothering. When we get to implementing it, it should be apparent what needs to change.

Reply:
On the other hand, maybe it would help whoever is doing the implementing avoid some weird bugs. I don't know.